vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SpreadsheetDropdownFactory.java

```diff
@@ -36,6 +39,14 @@ public Component getCustomEditorForCell(Cell cell,
     if (hasMoreThanOneValue(columnIndex) && !isHeaderRow(rowIndex)) {
       ComboBox<String> editorCombobox = createEditorCombobox(spreadsheet, cell.getColumnIndex(),
           cell.getRowIndex());
+      if (columnIndex == 1) {
```

Comment:
Is it always column index 1? how do we check for the other columns which information is contained?